### ISSUE TITLE
ARM Build

### ISSUE THREAD

**Issue by ltjax** [itchio/butler#204]
Hello,
I'm deploying to itch from my raspberry pi. So far, I compiled my own version of butler for this, but I'm getting errors like `Version check failed: HTTP 404: https://broth.itch.ovh/butler/linux-arm-head/LATEST`, and no being a Go person myself, it was quite an adventurous process to get there. It would be great to just get a compiled version from broth.

**Comment by fasterthanlime** [comment]
I don't really have access to any ARM machines, so there's no official builds.

I know the DragonRuby folks are doing ARM builds of butler though!

**Comment by fasterthanlime** [comment]
BTW the version check error should just be a warning right? It doesn't actually prevent butler from working?

**Comment by ltjax** [comment]
Yes, it seems to just be a warning. How are you building the other versions?
I think Travis CI supports ARM.

**Comment by neauoire** [comment]
We also work from solar/rpi, I would love an ARM build <3

**Comment by fasterthanlime** [comment]
I don't work at itch anymore — I think I remember others making butler builds for arm64, so it should be possible!

**Comment by neauoire** [comment]
Ah! Understood thanks for the head's up :) I'll try to get the golang toolchain going on the Pinebook.

**Comment by aidanoflann** [comment]
> Ah! Understood thanks for the head's up :) I'll try to get the golang toolchain going on the Pinebook.

Hey there - any progress on this? I'd also love an armv7 build of butler so I can use my raspberry pi 4 as a build machine!

**Comment by leafo** [comment]
@aidanoflann Did you try `go build` after checking out the repo. As far as I know the entire project should build on arm but I have not personally tested it

**Comment by aidanoflann** [comment]
> @aidanoflann Did you try `go build` after checking out the repo. As far as I know the entire project should build on arm but I have not personally tested it

Oh thanks! That was actually much more straightforward than I thought. As you suggested I just ran `go build` from the root dir of the repo.
